build-system/compile/bundles.config.bento.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "name": "amp-jwplayer",
+    "version": "1.0",
+    "latestVersion": "0.1",
+    "options": {
+      "hasCss": true,
+      "bento": true,
+      "npm": true
```

Comment:
maybe these will be redundant in the future since this is a specific json for bento